common/scala/src/main/scala/whisk/core/connector/Message.scala

```diff
@@ -56,7 +56,8 @@ case class ActivationMessage(
     activationNamespace: EntityPath,
     rootControllerIndex: InstanceId,
     content: Option[JsObject],
-    cause: Option[ActivationId] = None)
+    cause: Option[ActivationId] = None,
+    tracingMetadata: Option[Map[String, String]] = None)
```

Comment:
on the last point/question: https://github.com/apache/incubator-openwhisk/issues/254.
as part of replicating the controller, i expect redis will be in the mix and might double multiple duties.